# FinTrack - Personal Budget Tracker
**Course:** Data Analytics | **Date:** 5/10/2026

This notebook collects monthly income and expenses, then produces
a formatted budget summary - no libraries required. 

## Section 1: Analyst Information
Enter your name and the month being analyzed when prompted.

In [1]:
analyst = input("Analyst name: ")
month = input("Budget month: ")

Analyst name:  Cecilia
Budget month:  May 2026


## Section 2: Monthly Income and Expenses 
Enter each dollar ammount when prompted. Values are cast to float 
so arithmetic works correctly - no libraries needed.

In [6]:
income = float(input("Monthly income $: "))
rent = float(input("Rent / housing $: "))
food = float(input("Food / groceries $: "))
transport = float(input("Transport $: "))
utilities = float(input("Utilities $: "))
entertainment = float(input("Entertainmnet $: "))

Monthly income $:  4500
Rent / housing $:  600
Food / groceries $:  250
Transport $:  120
Utilities $:  100
Entertainmnet $:  65


## Section 3: Budget Calculations
Total expenses, remaining budget, and savings rate are computed using plain
Python arithmetic - no import statements required.

In [7]:
total_expenses = rent + food + transport + utilities + entertainment
remaining = income - total_expenses
savings_rate = (remaining / income) * 100

## Section 4A: Expense Breakdown by Category
A breakdown of each expense as a percentage of monthly income.


| Category | Amount | % of Income |   
|----------|--------|-------------|
| Rent/Housing | 600 | 13.3% |
| Food | 250 | 5.6% | 
| Transport | 120 | 2.7% |
| Utilities | 100 | 2.2% |
| Entertainment | 65 | 1.4% |


## Section 4B: Budget Summary Report
The report below is generated using Python f-strings.

In [8]:
print (f"""
====== FinTrack Budget Report ======
Analyst : {analyst}
Month   : {month}
-------------------------------------
Income           : ${income:,.2f}
Rent/Housing     : ${rent:,.2f}
Food/Groceries   : ${food:,.2f}
Transport        : ${transport:,.2f}
Utilities        : ${utilities:,.2f}
--------------------------------------
Total Expenses   : ${total_expenses:,.2f}
Remaining        : ${remaining:,.2f}
Savings Rate     : ${savings_rate:.1f}%
=======================================
""")


====== FinTrack Budget Report ======
Analyst : Cecilia
Month   : May 2026
-------------------------------------
Income           : $4,500.00
Rent/Housing     : $600.00
Food/Groceries   : $250.00
Transport        : $120.00
Utilities        : $100.00
--------------------------------------
Total Expenses   : $1,135.00
Remaining        : $3,365.00
Savings Rate     : $74.8%



## Budget Status
Evaluates the savings rate, categorizing it as **Healthy**, **Tight**, or 
**Over Buget** based on savings rate percentage.
- *Healthy*: equal to or over 20%
- *Tight*: 10 to 19%
- *Over Budget*: below 10%

In [9]:
if savings_rate >= 20:
    print(f"{analyst}'s budget status for {month} is Healthy.")
elif savings_rate >= 10:
    print(f"{analyst}'s budget status for {month} is Tight.")
else:
    print(f"{analyst}'s budget status for {month} is Over Budget.")


Cecilia's budget status for May 2026 is Healthy.
